### PR TITLE
chore: update yarn and missing deps

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Otherwise we will always forget these versions I think.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
     }
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale"
+  "vscode-vale.path": "${workspaceFolder}/node_modules/.bin/commercetools-vale",
+  "workbench.colorCustomizations": {
+    "tab.unfocusedActiveBorder": "#fff0"
```
Comment (emmenko) [inline .vscode/settings.json]: 👀

File: website-components-playground/package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
     "serve": "gatsby serve"
   },
   "dependencies": {
-    "@commercetools-frontend/application-components": "15.2.1",
-    "@commercetools-frontend/i18n": "15.4.1",
+    "@commercetools-frontend/application-components": "15.4.2",
```
Comment (emmenko) [inline website-components-playground/package.json]:
```suggestion
    "@commercetools-frontend/application-components": "15.5.0",
```

File: website-components-playground/package.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
...
+    "@commercetools-frontend/i18n": "15.4.2",
```
Comment (emmenko) [inline website-components-playground/package.json]:
```suggestion
    "@commercetools-frontend/i18n": "15.5.0",
```

File: .vscode/settings.json (hunk 1 above)
Comment (tdeekens) [reply .vscode/settings.json]: Gaar. Forgot to push that commit. I start to dislike this file :)

Comment (emmenko) [reply .vscode/settings.json]: If it gets too annoying, we can ignore it again.

File: website-components-playground/package.json (hunk 2 above)
Comment (tdeekens) [reply website-components-playground/package.json]: Good catch. I just went with latest.

Comment (emmenko) [reply website-components-playground/package.json]: Yeah, we can actually promote the packages to latest. Will do that now

**Comment by tdeekens** [comment]
Thanks @emmenko messed up using latest over next. Have another look :)

**Comment by emmenko** [comment]
The vscode setting?

**Comment by tdeekens** [comment]
Gaaaar. Soz. It's very persistent in re-adding it. Need to remove that plugin I suppose.

**Comment by emmenko** [comment]
Thanks

**Comment by tdeekens** [comment]
Zeit failed again but it seems on their side. I can't restart the build. I think I will merge and see what happens on master.

**Comment by emmenko** [comment]
The error from Zeit:

image

**Pull request merged**